lib/JavascriptGenerator.js

```diff
@@ -0,0 +1,185 @@
+/*
+	MIT License http://www.opensource.org/licenses/mit-license.php
+	Author Tobias Koppers @sokra
+*/
+"use strict";
+
+const RawSource = require("webpack-sources").RawSource;
+const ReplaceSource = require("webpack-sources").ReplaceSource;
+
+// TODO: clean up this file
+// replace with newer constructs
+
+// TODO: remove DependencyVariables and replace them with something better
+
+class JavascriptGenerator {
```

Comment:
JavaScript is written with a capital S. Every time I see it written this way in webpack I need to breath deeply 😅

Reply:
PascalCase ignores propernow capitalization I believe. Since this is only a two word name.

Reply:
So, can we talk about WebAssembly? 😉

Reply:
Touche. Haha well we should just update them all in one PR, not this one though. 😂